packages/docs/site/docs/08-query-api/01-index.md

```diff
@@ -27,6 +27,7 @@ You can go ahead and try it out. The Playground will automatically install the t
 | `wp`           | `latest`      | Loads the specified WordPress version. Supported values: `5.9`, `6.0`, `6.1`, `6.2`, `latest`                                                                                                                                                                                                                                                                  |
 | `plugin`       |               | Installs the specified plugin. Use the plugin name from the plugins directory URL, e.g. for a URL like `https://wordpress.org/plugins/wp-lazy-loading/`, the plugin name would be `wp-lazy-loading`. You can pre-install multiple plugins by saying `plugin=coblocks&plugin=wp-lazy-loading&…`. Installing a plugin automatically logs the user in as an admin |
 | `theme`        |               | Installs the specified theme. Use the theme name from the themes directory URL, e.g. for a URL like `https://wordpress.org/themes/disco/`, the theme name would be `disco`. Installing a theme automatically logs the user in as an admin                                                                                                                      |
+| `import`       |               | Imports a site from the specified zip file URL. The file must be served with CORS enabled. The step automatically logs the user in as an admin, unless the default user has been changed or option `login` is set to `0` (zero)                                                                                                                                |
```

Comment:
> the file must be served with appropriate CORS headers (must allow fetching from the domain on which the Playground is running).

we might want to expand this, because "CORS enabled" may not say much. I _think_ the most important part is that a site allows the Playground domain as an allowable origin if using `Access-Control-Allow-Origin`.

Reply:
> The step automatically logs the user in as an admin, unless the default user has been changed or option `login` is set to `0` (zero)

this is great, and I'm glad you put it in, but don't we need to update our code to do this too?

if we don't want to modify the code at this time given the question it raises "how do we know if the login will succeed before trying?" then we could also note here something like "When importing a site, no auto-login will be performed because there's no guarantee that the admin user is the same as the default one set in the Playground. To auto-login a user add a `login` step in blueprints (see [[Blueprints in the hash]])"